### PR TITLE
mayhem integration

### PR DESCRIPTION
**Pull request by evanrichter**

### CODE REVIEW

File: test/fuzz/mayhem/lwip.mayhemfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+project: lwip-tcpip
```
Comment (xansec) [inline test/fuzz/mayhem/lwip.mayhemfile:1]: Are all of these the same target? If not, they should be split into multiple Mayhemfiles or omitted.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: test/fuzz/mayhem/lwip.mayhemfile (hunk 1 above)
Comment (evanrichter) [reply test/fuzz/mayhem/lwip.mayhemfile:1]: they are the same target, the 2 and 3 refer to the number of packets to attempt to parse. In my opinion they should all share the same corpus

**Comment by xansec** [comment]
gotcha, lgtm!

**Pull request merged**